e2e_playwright/st_dataframe_interactions_test.py

```diff
@@ -558,6 +558,20 @@ def test_row_hover_highlight(themed_app: Page, assert_snapshot: ImageCompareFunc
     assert_snapshot(df, name="st_dataframe-row_hover_highlight")
 
 
+def test_autosize_column_via_ui(app: Page, assert_snapshot: ImageCompareFunction):
+    """Test that a column can be autosized via the UI via the column menu."""
+    df = app.get_by_test_id("stDataFrame").nth(0)
+    expect_canvas_to_be_visible(df)
+
+    # Open the column menu of the index column and autosize the column:
+    open_column_menu(df, 0, "small")
+    app.get_by_test_id("stDataFrameColumnMenu").get_by_text("Autosize").click()
+    unfocus_dataframe(app)
+    # Use the same screenshots as above since we expect the same
```

Comment:
Which screenshot as above? I did not find any screenshot with the same name

Reply:
Oh, sorry... the comment is wrong here :(

Reply:
I added a small improvement to the test to ensure that the new dataframe is smaller than the old one